redirects-www.conf

```diff
@@ -86,6 +86,7 @@ rewrite ^/press/weekly_digests.shtml /updates/?update_type=weekly-digest redirec
 rewrite ^/press/campaign_finance_statistics.shtml https://transition.fec.gov/press/campaign_finance_statistics.shtml redirect;
 rewrite ^/press/resources_for_reporters.shtml /press/resources-journalists/ redirect;
 rewrite ^/press/press_contacts.shtml /press/#contact redirect;
+rewrite ^/press/archive/(.*) /resources/news_releases/$1 redirect;
```

Comment:
@johnnyporkchops Since nginx processes redirects  based on first match, I like to put the more generic redirects at the bottom of the file, that way we can override generic redirects with specific absolute redirects if needed. Generic redirects start on line 136, so please move this generic redirect there.